tests/aws_iot_secure_tunneling_client_test.c

```diff
@@ -21,27 +21,61 @@ static void s_on_connection_complete(const struct aws_secure_tunnel *secure_tunn
     aws_mutex_unlock(&mutex);
 }
 
+static void s_on_data_receive(const struct aws_secure_tunnel *secure_tunnel, const struct aws_byte_buf *data) {
+    UNUSED(secure_tunnel);
+
+    printf("Client received data:");
+    /* Didn't want to copy to a null terminated string. So just print out each character */
+    for (size_t i = 0; i < data->len; i++) {
+        printf("%c", data->buffer[i]);
+    }
+    printf("\n");
+}
+
+static void s_on_stream_start(const struct aws_secure_tunnel *secure_tunnel) {
+    printf("Client received StreamStart. streamId=%d\n", secure_tunnel->stream_id);
+}
+
+static void s_on_stream_reset(const struct aws_secure_tunnel *secure_tunnel) {
+    UNUSED(secure_tunnel);
+    printf("Client received StreamReset.\n");
+}
+
+static void s_on_session_reset(const struct aws_secure_tunnel *secure_tunnel) {
+    UNUSED(secure_tunnel);
+    printf("Client received SessionReset.\n");
+}
+
+enum aws_secure_tunneling_local_proxy_mode s_local_proxy_mode_from_c_str(const char *local_proxy_mode) {
```

Comment:
comment: tokens will start with `src-` or `dest-`

or something similar

Reply:
This is to convert a CLI argument into local proxy mode (source vs. destination), not access token.

Reply:
Oh ok, the call is on a variable called token, so I assumed ;)